app/assets/javascripts/config.js.coffee.erb

```diff
@@ -13,6 +13,9 @@
   match = /cmr_env=(\w+)/.exec(query)
   cmr_env = if match then match[1] else '<%= Rails.configuration.cmr_env %>'
 
+  
+  reverb_url = "<%= ENV['reverb_url']%>"
```

Comment:
Grab a value from services.yml.erb as the default. You can refer to how borwse_scaler_url and other variables are set up below.